source/agora/consensus/EnrollmentManager.d

```diff
@@ -489,6 +489,50 @@ public class EnrollmentManager
         return makeEnrollment(kp, utxo, cycle_length, cache[$ - offset - 1], offset);
     }
 
+    version (unittest)
+    private static void checkGenesisEnrollments(const Block genesisBlock, KeyPair[] keys, uint cycle_length)
```

Comment:
Maybe move to `genesis/package.d` along with `makeGenesis`, since they seem to be aimed at the same purpose ?
Also, style (space before parenthesis)

Reply:
Moved and corrected.